Packs/Azure/Integrations/Azure/Azure.yml

```diff
@@ -0,0 +1,1734 @@
+      type: String
+    - contextPath: Azure.KeyVault.type
+      description: Resource type in Azure.
+      type: String
+    - contextPath: Azure.KeyVault.location
+      description: Key Vault location.
+      type: String
+    - contextPath: Azure.KeyVault.tags
+      description: Resource tags.
+    - contextPath: Azure.KeyVault.properties.sku.family
+      description: SKU family name.
+      type: String
+    - contextPath: Azure.KeyVault.properties.sku.name
+      description: SKU name to specify whether the key vault is a standard vault or a premium vault.
+      type: String
+    - contextPath: Azure.KeyVault.properties.tenantId
+      description: The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault.
+      type: String
+    - contextPath: Azure.KeyVault.properties.accessPolicies
+      description: An array of 0 to 16 identities that have access to the key vault. All identities in the array must use the same tenant ID as the key vault's tenant ID.
+    - contextPath: Azure.KeyVault.properties.accessPolicies.tenantId
+      description: The Azure Active Directory tenant ID that should be used for authenticating requests to the key vault.
+      type: String
+    - contextPath: Azure.KeyVault.properties.accessPolicies.objectId
+      description: The object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The object ID must be unique for the list of access policies.
+      type: String
+    - contextPath: Azure.KeyVault.properties.accessPolicies.permissions
+      description: Permissions the identity has for keys, secrets and certificates.
+    - contextPath: Azure.KeyVault.properties.enabledForDeployment
+      description: Property to specify whether Azure Virtual Machines are permitted to retrieve certificates stored as secrets from the key vault.
+      type: Boolean
+    - contextPath: Azure.KeyVault.properties.enabledForDiskEncryption
+      description: Property to specify whether Azure Disk Encryption is permitted to retrieve secrets from the vault and unwrap keys.
+      type: Boolean
+    - contextPath: Azure.KeyVault.properties.enabledForTemplateDeployment
+      description: Property to specify whether Azure Resource Manager is permitted to retrieve secrets from the key vault.
+      type: Boolean
+    - contextPath: Azure.KeyVault.properties.enableSoftDelete
+      description: Property to specify whether the 'soft delete' functionality is enabled for this key vault.
+      type: Boolean
+    - contextPath: Azure.KeyVault.properties.enablePurgeProtection
+      description: Property to specify whether the 'purge protection' functionality is enabled for this key vault.
+      type: Boolean
+    - contextPath: Azure.KeyVault.properties.enableRbacAuthorization
+      description: Property to specify whether Azure Key Vault uses Role Based Access Control (RBAC) for authorization of data actions.
```

Comment:
Whether Azure Key Vault uses Role Based Access Control (RBAC) for authorization of data actions.